react-frontend/src/Components/Map/Map.js

```diff
@@ -14,26 +14,30 @@ L.Icon.Default.mergeOptions({
 });
 
 
-const filteredStations = teslaData.filter(tesla => tesla.address.country === "Italy")
+const filteredStations = data.filter(data => (data.utilitiesIncluded == true) && data.furnitureProvided == true)
 function Map() {
-  console.log(teslaData);
+  console.log(data);
 
   return (
-    <MapContainer center={[42.58544, 13.257684]} zoom={6} scrollWheelZoom={true}>
+    <MapContainer center={[43.47221, -80.54474]} zoom={15} scrollWheelZoom={true}>
       <TileLayer
         attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
         url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
       />
 
-      {filteredStations.map(tesla => (
+      {filteredStations.map(location => (
         <Marker 
-        key = {tesla.id}
-        position={[ tesla.gps.latitude, tesla.gps.longitude]}>
+        key = {location.id}
+        position={[ location.gps.latitude, location.gps.longitude]}>
 
-          <Popup position={[ tesla.gps.latitude, tesla.gps.longitude]}>
+          <Popup position={[ location.gps.latitude, location.gps.longitude]}>
             <div>
-              <h2>{"Name:" + tesla.name}</h2>
-              <p>{"Status:" + tesla.status}</p>
+              <h2>{location.name}</h2>
```

Comment:
Something interesting you can look into is may put optional chaining here. It's a new feature of javascript added in 2020. It could be combined with ternary operator to handle a object field that isn't there and could cause an error